src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -4,6 +4,8 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_GENDER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_JOBID;
```

Comment:
Just a minor naming consideration, but I think it is better to add space to this variable, i.e. `PREFIX_JOB_ID`